### PR TITLE
Add MongoDB deployment and service.

### PR DESCRIPTION
**Pull request by Shubham82**
Add MongoDB deployment and service in `/examples/application/mongodb` and change their links in `Use Port Forwarding to Access Applications in a Cluster` task.

Fix: #28500

### CODE REVIEW

**Comment by Shubham82** [comment]
Hi @daminisatya @zparnold
Could please take a look!

**Comment by tengqm** [comment]
@jimangel Please chime in whether adding mongodb as an example is a good choice. I noticed that you were replacing the mongodb by redis in another example.

**Comment by jimangel** [comment]
Thanks for flagging this @tengqm and thanks for the PR @Shubham82! Looking at the PR history, I see that it [used to be `redis`](https://github.com/jihoon-seo/k8s-website/blob/e651788efd11c9cc708ea6809f7b7367dabef15c/content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md) and that was [later updated to mongoDB](https://github.com/kubernetes/website/commit/9068e7071685a8db58076272f45c2d99b58fbd17#diff-f4812cdf3b967068bcd2c8b1ed666debf27703fc2e85cdd8cef5ee9869a3cfe9).

This must have been before we realized that the examples were borrowed from a Google repo and it makes more sense to use their source vs. maintaining our own. As a result, I started to PR the changes. For more info: https://github.com/kubernetes/website/pull/28452

To be extremely explicit, this isn't about picking a preferred technology, it's about leveraging existing sourced content. I would suggest either / or:
- closing this PR
- modifying it to match some of the deployments from: https://github.com/kubernetes/website/pull/28452/files.

Not a requirement, but you can also comment out the source of the yaml at the top of the file like in [this example](https://github.com/jimangel/website/blob/ff0018b97e68e6bd6f0fb8323b9508a6c14d83de/content/en/examples/application/guestbook/frontend-deployment.yaml#L1).

**TL;DR:** This PR saw a perfect storm of being the "right" fix at the wrong time; given the other PRs in flight.

Thanks again! Let me know if I can clarify anything!

**Comment by Shubham82** [comment]
Thanks, @tengqm @jimangel for taking look at this.
@tengqm  can you suggest to me what to do now. either uses Redis(which one from [this](https://github.com/kubernetes/website/tree/main/content/en/examples/application/guestbook)) instead of MongoDB or uses MongoDB for this task.

**Comment by tengqm** [comment]
According to @jimangel 's clarification, we do not have any problems running MongoDB or Redis. I think this one is good to go as a simpler fix when compared to switching to other services, which means the whole page needs to be reworked.

/lgtm

**Comment by Shubham82** [comment]
thanks, @tengqm for your response.

**Comment by Shubham82** [comment]
Hi @savitharaghunathan
Could please take a look

Thanks!

**Comment by Shubham82** [comment]
Hi @tengqm could you please approve this request.

**Comment by Shubham82** [comment]
Hi @tengqm  @savitharaghunathan 
could anyone please approve this PR so that it will merge soon, as so many issues related to it are opened! 
Thanks
